python/src/scipp/plotting/toolbar.py

```diff
@@ -42,8 +43,24 @@ def __init__(self, mpl_toolbar=None):
         self.mpl_toolbar = mpl_toolbar
 
         self.add_button(name="home_view", icon="home", tooltip="Reset original view")
+        self._resampling_mode = _make_toggle_button(
+            description='',
+            tooltip="Switch current resampling mode. Options are 'sum' and 'mean'."
+            " The correct mode depends on the interpretation of data."
+            " The default value is guessed based on the data unit but in practice"
+            " the automatic selection cannot always be relied on.")
 
     def initialize(self, log_axis_buttons, button_states):
+        self.members['resampling_mode'] = self._resampling_mode
```

Comment:
Something going wrong here? Button set to toggled by default if we have `mean`, but not if we set `sum`.

![image](https://user-images.githubusercontent.com/1109536/133056442-2fb2b80d-f7b2-43da-9474-a7641b076c55.png)

Reply:
Not sure I understand your comment. We have two states, 'sum' and 'mean', and depict 'sum as "button off" and 'mean' as "button on".

Reply:
The combination of a single button doing the toggle and the changing text feels slightly weird. However, I'm not sure there is a better way to achieve this with a single button representing two states that cannot self-describe their inverse state.